src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidatorTestForInvalidFiles.java

```diff
@@ -0,0 +1,139 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.validation;
+
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.io.IOUtils;
+import org.assertj.core.api.AbstractThrowableAssert;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.springframework.boot.test.rule.OutputCapture;
+import uk.gov.hmcts.reform.bulkscanprocessor.exceptions.InvalidEnvelopeSchemaException;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+public class MetafileJsonValidatorTestForInvalidFiles {
+
+    @Rule
+    public OutputCapture capture = new OutputCapture();
+
+    private MetafileJsonValidator validator;
+
+    @Before
+    public void setUp() throws IOException, ProcessingException {
+        validator = new MetafileJsonValidator();
+    }
+
+    @After
+    public void tearDown() {
+        capture.flush();
+    }
+
+    @Test
+    public void should_not_parse_envelope_with_no_scannable_items_in() {
+        tryToGetEnvelope(
```

Comment:
I think these test cases are not easily readable. what do you think of below approach?

```@Test
    public void should_not_parse_envelope_with_no_scannable_items_in() throws Exception {
        //Given
        byte[] noScannableMetaFile = IOUtils.toByteArray(getClass().getResource("/metafiles/invalid/no-scannables.json"));

        //when
        AbstractThrowableAssert actualAssertion = assertThatCode(() -> validator.validate(noScannableMetaFile));

        //then
        actualAssertion.isInstanceOf(InvalidEnvelopeSchemaException.class);

        actualAssertion.hasMessage("Failed validation against schema:\n" +
            "\tobject has missing required properties ([\"scannable_items\"])");

    }
```

I know you have tried to extract common code into a method but that's making it a bit difficult to read.

Reply:
i think it's a lot of repetitive work and will put developers to sleep while scrolling. also `AbstractThrowableAssert` has self type check which is kind-of weird and i didn't check how to correctly assign it as a variable.

that said, i can still convert everything how you suggest